docs/cep/examples/table-and-window-example.md

```diff
@@ -52,5 +52,5 @@ FROM InboundTraffic WINDOW SLIDING_TIME(2 sec) as IT
 JOIN SuspiciousIPTable as SIP
 ON  IT.ip == SIP.blocked_ip
 GROUP BY ip
-HAVING requestCount > 50 ;
```

Comment:
@diana-macrometa - Incorrect, after `HAVING` always follows `GROUP BY` but not `WHERE`.

Reply:
Okay, I think I get it.

Reply:
@diana-macrometa - Let me double check it.

Reply:
@diana-macrometa - The compiler allows both `where | having` after `group by`. 
However in SQL it's `having` rather than `where`.

Reply:
May be that way in SQL, but Macrometa changed HAVING to WHERE, according to the Migration Guide for QL v2. 

Do we not want to use WHERE? Just reading it, it makes a ton more sense than HAVING, so I would prefer WHERE.

Reply:
@diana-macrometa - If I'm a customer and I'm familiar with SQL, construction like this below will look rather strange to me and uncommon.
```
SELECT product_id, count() AS amount
FROM products
WHERE product_id > 1000
GROUP BY product_id
WHERE count() > 10
```
instead of 
```
SELECT product_id, count() AS amount
FROM products
WHERE product_id > 1000
GROUP BY product_id
HAVING count() > 10
```

Reply:
No. `WHERE ... GROUP BY ...` is not equivalent to `GROUP BY ... WHERE | HAVING ...`.
E.g. in my sample above the count() is performed first on a group and then have to be filtered.
So, let's revert back the previous version but keep the `HAVING` after the `GROUP BY`.

Reply:
Should I close the entire PR, or just revert this one piece?

Reply:
Yes, makes sense.

Reply:
Which option makes sense? Close the PR? Or revert this one part?

Reply:
Please close the PR and create another with the fix only in `docs/cep/windows/window-types/tumbling-length.md`. Let's keep `HAVING` into the other files.